pkg/api/helpers.go

```diff
@@ -73,3 +74,11 @@ var standardResources = util.NewStringSet(
 func IsStandardResourceName(str string) bool {
 	return standardResources.Has(str)
 }
+
+func ProbeTimeout(container Container) time.Duration {
```

Comment:
This should accept a `api.Probe` and live in `pkg/probe/probe.go` until defaults are centralized. Two reasons: We are trying to decouple probes from api objects like Container, and `container.LivenessProbe.TimeoutSeconds` looks like it could through a nil pointer dereference panic if this function is used incorrectly. Optionally, `1 * time.Second` could live in a nicely named const.